controllers/nodehealthcheck_controller.go

```diff
@@ -131,8 +165,9 @@ func (r *NodeHealthCheckReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 
 	// check if we need to disable NHC because of missing template CR
 	var template *unstructured.Unstructured
-	if template, err = r.fetchTemplate(nhc); err != nil {
+	if template, err = resourceManager.GetTemplate(nhc); err != nil {
 		if !utils.IsConditionTrue(nhc.Status.Conditions, remediationv1alpha1.ConditionTypeDisabled, remediationv1alpha1.ConditionReasonDisabledTemplateNotFound) {
+			log.Info("disabling NHC, template not found")
```

Comment:
NIT: Maybe `disabling NHC, since template was not found` or `disabling NHC - template was not found`?